examples/SAP-to-Database/Ballerina.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+org = "integration_samples"
```

Comment:
```suggestion
org = "wso2"
```

Reply:
Resolved here [Improve SAP to DB example](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/23/commits/e6de654368da7d53a4bfc51ded0122365908f2b2)